### PR TITLE
ci(schema-wasm): cleanup the GitHub Action

### PR DESCRIPTION
**Pull request by Jolg42**
- auto formatting
- add run-name
- switch to Node.js 20
- use `registry-url`
- split some commands into multiple standalone ones

It makes it similar to the other publishing actions we have.

### CODE REVIEW

File: .github/workflows/publish-prisma-schema-wasm.yml [hunk 1]
```diff
@@ -30,26 +31,19 @@ jobs:
...
-      - name: Set up NPM token
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+      - name: Update version in package.json
+        run: PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
 
-      - run: |
-          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
-          npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
+      - name: Publish Prisma Driver Adapters packages
+        run: npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply .github/workflows/publish-prisma-schema-wasm.yml:46]: The only thing wrong is the text

File: .github/workflows/publish-prisma-schema-wasm.yml [hunk 2]
```diff
@@ -30,26 +31,19 @@ jobs:
...
 
-      - name: Set up NPM token
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+      - name: Update version in package.json
+        run: PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
 
-      - run: |
-          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
-          npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
+      - name: Publish Prisma Driver Adapters packages
```
Comment (Jolg42) [inline .github/workflows/publish-prisma-schema-wasm.yml:45]:
```suggestion
      - name: Publish @prisma/prisma-schema-wasm
```

**Review by miguelff — APPROVED** [review]

**Comment by Jolg42** [comment]
@janpio I can't merge because you blocked the PR

File: .github/workflows/publish-prisma-schema-wasm.yml [hunk 3]
```diff
@@ -30,26 +31,19 @@ jobs:
...
-      - name: Set up NPM token
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+      - name: Update version in package.json
+        run: PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
 
-      - run: |
-          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
-          npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
+      - name: Publish @prisma/prisma-schema-wasm
+        run: npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
```
Comment (janpio) [inline .github/workflows/publish-prisma-schema-wasm.yml:46]: Does `PACKAGE_DIR=...` set the env variable for the next `run` in the next step as well?

Comment (Jolg42) [reply .github/workflows/publish-prisma-schema-wasm.yml:46]: Right... Thanks!

Comment (Jolg42) [reply .github/workflows/publish-prisma-schema-wasm.yml]: It's definitely weird, but fixed now

File: .github/workflows/publish-prisma-schema-wasm.yml [hunk 4]
```diff
@@ -30,24 +31,16 @@ jobs:
...
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org/'
 
-      - name: Set up NPM token
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
-
-      - run: |
+      - name: Update version in package.json & Publish @prisma/prisma-schema-wasm
+        run: 
           PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
```
Comment (janpio) [inline .github/workflows/publish-prisma-schema-wasm.yml:44]:
```suggestion
          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }}) # Update version in package.json and return directory for later usage
```

**Review by janpio — APPROVED** [review]

File: .github/workflows/publish-prisma-schema-wasm.yml [hunk 5]
```diff
@@ -30,25 +31,17 @@ jobs:
...
+          registry-url: 'https://registry.npmjs.org/'
 
-      - name: Set up NPM token
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
-
-      - run: |
-          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
+      - name: Update version in package.json & Publish @prisma/prisma-schema-wasm
+        run: 
+          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }}) # Update version in package.json and return directory for later usage
```
Comment (Jolg42) [inline .github/workflows/publish-prisma-schema-wasm.yml:44]:
```suggestion
          # Update version in package.json and return directory for later usage
          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }}) 
```

**Pull request merged**